@xylar, I'm seeing an error when I deploy on Frontier
```
brus@login07:~/project/polaris_worktrees/main$ ./deploy.py 

 Running from:
   /autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/deploy_tmp/bootstrap_pixi
 Running:
   /ccs/home/brus/.pixi/bin/pixi install

✔ The default environment has been installed.
Loaded hooks from /autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/deploy/hooks.py: pre_pixi, pre_spack
Running hook stage=pre_pixi file=/autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/deploy/hooks.py func=pre_pixi
Wrote hook context snapshot stage=pre_pixi path=/autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/deploy_tmp/hooks/pre_pixi_context.json

 Running from:
   /autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/pixi-env
 Running:
   /ccs/home/brus/.pixi/bin/pixi install

✔ The default environment has been installed.
Using cached JIGSAW build

 Running from:
   /autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main
 Running:
   env -u PIXI_PROJECT_MANIFEST -u PIXI_PROJECT_ROOT -u PIXI_ENVIRONMENT_NAME -u PIXI_IN_SHELL /ccs/home/brus/.pixi/bin/pixi workspace channel add --manifest-path /autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/pixi-env/pixi.toml --prepend file:///autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/.mache_cache/jigsaw

✔ Added file:///autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/.mache_cache/jigsaw

 Running from:
   /autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main
 Running:
   env -u PIXI_PROJECT_MANIFEST -u PIXI_PROJECT_ROOT -u PIXI_ENVIRONMENT_NAME -u PIXI_IN_SHELL /ccs/home/brus/.pixi/bin/pixi add --manifest-path /autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/pixi-env/pixi.toml --platform linux-64 'jigsawpy=1.1.0.*'

✔ Added jigsawpy=1.1.0.*
Added these only for platform(s): linux-64
Running hook stage=pre_spack file=/autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/deploy/hooks.py func=pre_spack
Wrote hook context snapshot stage=pre_spack path=/autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/deploy_tmp/hooks/pre_spack_context.json

 Running from:
   /autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main
 Running:
   eval "$(env -u PIXI_PROJECT_MANIFEST -u PIXI_PROJECT_ROOT -u PIXI_ENVIRONMENT_NAME -u PIXI_IN_SHELL /ccs/home/brus/.pixi/bin/pixi shell-hook -s bash -m /autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/pixi-env/pixi.toml)"
   pip install --no-deps --no-build-isolation -e .

/bin/bash: /autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/pixi-env/.pixi/envs/default/bin/pip: /autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/polaris/pixi-env/.pixi/envs/default/bin/python3.14: bad interpreter: No such file or directory
Traceback (most recent call last):
  File "/autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/deploy_tmp/bootstrap_pixi/.pixi/envs/default/bin/mache", line 8, in <module>
    sys.exit(main())
             ~~~~^^
  File "/autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/deploy_tmp/bootstrap_pixi/.pixi/envs/default/lib/python3.14/site-packages/mache/__main__.py", line 21, in main
    args.func(args)
    ~~~~~~~~~^^^^^^
  File "/autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/deploy_tmp/bootstrap_pixi/.pixi/envs/default/lib/python3.14/site-packages/mache/deploy/cli.py", line 91, in _dispatch_deploy
    run_deploy(args=args)
    ~~~~~~~~~~^^^^^^^^^^^
  File "/autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/deploy_tmp/bootstrap_pixi/.pixi/envs/default/lib/python3.14/site-packages/mache/deploy/run.py", line 357, in run_deploy
    _install_software_in_dev_mode(
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        pixi_exe=pixi_exe,
        ^^^^^^^^^^^^^^^^^^
    ...<2 lines>...
        quiet=quiet,
        ^^^^^^^^^^^^
    )
    ^
  File "/autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/deploy_tmp/bootstrap_pixi/.pixi/envs/default/lib/python3.14/site-packages/mache/deploy/run.py", line 1826, in _install_software_in_dev_mode
    check_call(
    ~~~~~~~~~~^
        cmd_install_software,
        ^^^^^^^^^^^^^^^^^^^^^
        log_filename=log_filename,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
        quiet=quiet,
        ^^^^^^^^^^^^
    )
    ^
  File "/autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/deploy_tmp/bootstrap_pixi/.pixi/envs/default/lib/python3.14/site-packages/mache/deploy/bootstrap.py", line 149, in check_call
    raise subprocess.CalledProcessError(
        process.returncode, commands, output=stdout_data
    )
subprocess.CalledProcessError: Command 'eval "$(env -u PIXI_PROJECT_MANIFEST -u PIXI_PROJECT_ROOT -u PIXI_ENVIRONMENT_NAME -u PIXI_IN_SHELL /ccs/home/brus/.pixi/bin/pixi shell-hook -s bash -m /autofs/nccs-svm1_proj/cli115/sbrus/polaris_worktrees/main/pixi-env/pixi.toml)" && pip install --no-deps --no-build-isolation -e .' returned non-zero exit status 126.

ERROR: Deployment step failed (exit code 1). See the error output above.
```

--- a/deploy/cli_spec.json
+++ b/deploy/cli_spec.json
@@ -1,7 +1,7 @@
 {
   "meta": {
     "software": "polaris",
-    "mache_version": "3.5.2",
+    "mache_version": "3.6.0",
     "description": "Deploy polaris environment"
   },
   "arguments": [

--- a/deploy/pins.cfg
+++ b/deploy/pins.cfg
@@ -3,7 +3,7 @@
 bootstrap_python = 3.14
 python = 3.14
 geometric_features = 1.6.3
-mache = 3.5.2
+mache = 3.6.0
 mpas_tools = 1.5.0
 otps = 2021.10
 parallelio = 2.6.9

